### ISSUE TITLE
CinnVIIStarkMenu@NikoKrause - Unable to read search results within cinnviistarkmenu in dark theme and cant find where to mod

### ISSUE THREAD

**Issue by qoaa** [linuxmint/cinnamon-spices-applets#1395]
Applet UUID - CinnVIIStarkMenu@NikoKrause
 * Applet version/Build date: 9/23/2017 20:55:13 UTC
 * Cinnamon version: 3.4.6
 * Distribution - Linux Mint 18.2 
 * Graphics:  Card: NVIDIA G92 [GeForce 9800 GTX+]
           Display Server: X.Org 1.18.4 driver: nvidia Resolution: 1600x900@60.00hz
           GLX Renderer: GeForce 9800 GTX+/PCIe/SSE2 GLX Version: 3.3.0 NVIDIA 340.102

**Short summary of system which for some odd reason doesnt list my ssd, or my 3 external drives .**
http://image.dude-suit.net/basicsystemsummary.png

 *** Snippet or attachment of ~/.cinnamon/glass.log:** http://image.dude-suit.net/cinnamon-glasslog.txt
 *** Snippet or attachment of ~/.xsession-errors:** http://image.dude-suit.net/xsession-errorlog.txt

**Regarding Applet:** CinnVIIStarkMenu

Steps to reproduce
Wanted to try the CinnVIIStarkMenu, it's design in for me personally is great. And all works perfectly, But I'd never really used the search bar but thought I would and noticed it's nearly unreadable on a dark theme. Or at least a semi dark not necessarily a all black based theme. I spent a couple hours trying to modify the search result text color myself using command line in one side and clicking the menu over and over in the other side as I'd adjust or at least test what color settings a specific menu is set at and copying into gcolor2 to see the color, which after finding Gcolor2 saved me so much time but still I couldn't locate specifically where the search results text color is set.

Here is 3 screenshots showing the example of the near unreadable text when using the search bar inside of the menu, one with search results only, second with near unreadable results on the left above search bar but all other icons and text on the right menu is perfect. I've found no theme issues in any of the options or submenus except the search results using the in menu search bar. 

**Screenshot 1:** http://image.dude-suit.net/cinviistartmenu-1.png

**Screenshot 2:** http://image.dude-suit.net/cinviistartmenu-2.png
this one shows it's only the search result list on left seems to be the unreadable one, all else to me, that is, is fine. 

**Screenshot 3:** This shows an example how I was going through anything to see if  I could luck across a match trying to search "search" in the script but no luck so far but Gcolor2 does speed up testing

http://image.dude-suit.net/cinviistartmenu-researching.png

******Other information**

I know things are easier said than done, but it would be really cool if maybe there was a menu setting so to speak that had like basic/advanced toggle that opened up allowing to manually adjust tint/text color/highlight or whatnot. For ease that is. 

Regardless if someone could give me a filename or location, and either line number, variable, etc to use find to get to it and then i can backup original and edit the color codes for the search result list. 

or perhaps ive got something mistaken or screwy on my system, either way I've spent couple evenings for a little while at least on my own trying to find it and something tells me it's right in front of my nose lol

**_Thanks for anyone's time, and major kudos to the developers!!!_**

**Comment by jaszhix** [comment]
Ping @NikoKrause. I'm a little confused, are you talking about the category texts?

**Comment by qoaa** [comment]
Yea the left side of menu, notice in screen 1 and screen 2 on the left inside the menu.

one is a search result typing in search bar on bottom left. the next is categories on left.

ive sent through all the configuration settings and cant seem to get the items appearing on left of menu above search bubble to he very readable. see how on the right it's nice crisp sharp darker easier to read but both search and categories on left seem as if to be faded or males me feel there's a hidden transparency slider in a sense. hehe

again not a major issue, just strange I can't find a way to darken or change color, etc of search results or categories.

do you see what I'm talking about? course full honesty I do wear glasses but to me the left pane of the menu text is faded.

But I just wanted to mention and ask and suggest something as an addon feature or maybe where can I adjust opaqueness or color of text and categories on left side. The text results from search are worse as screenshot 1 and 2 show.

**Comment by NikoKrause** [comment]
I guess you are using the Carta theme.

The search results are perfectly readable. What you describe to be unreadable are the categories (Games, Internet, etc.) during the search. 

First: It's not a CinnVIIStarkMenu issue only. It looks the same with the default Cinnamon menu.
Second: It's not the problem of a menu to deal with themeing. Colors are handled by the theme. Thus it should be fixed by the theme.
Third: The categories are greyed out during the search to indicate, that you can't click those buttons during the search. 

If you think it's unreadable, you probably should get in touch with the creator of the theme. The [Carta theme](https://cinnamon-spices.linuxmint.com/themes/view/Carta) is maintained by [andr35](https://github.com/andr35).

If you want to change the theme on your own (I think you searched for something like `search`), since those are the category buttons, which are greyed out, you can change those by changing the `cinnamon.css` theme, namely the lines:
```
.menu-category-button-greyed {
    color: rgba(250, 0, 0, 0.45);
}
```
The changes you make will only apply after restarting Cinnamon (CTRL+ALT+ESC).

But remember, that your changes will be overwritten, if you update the theme.

**Comment by NikoKrause** [comment]
P.S.: I'm happy that you like the theme and I hope this solves your issue.

**Comment by qoaa** [comment]
Thank you very much! I wasn't sure if it was the applet or my desktop theme that might have been getting in the way. I started wondering if my eyes were going bad, I already wear glasses as it is... haha

But I do appreciate your time with this, that css line mod did solve it all for me. I was digging in wrong place in the css, Trying to use gcolor2 to copy in the css color codes to see if I could stumble across which was responsible for it, but still it was a fun time killer anyhow when I didn't have anything to do really.

And yea when I was using 'nano' to dig and search css (extremely old habit, as back after getting cs degree, and a couple certs, I'd always be found on our campus library system doing email in pine and editing files lazily in pico lol).

But you are spot on, I was searching for "search" in the code and trying to locate the rgb values there. 

I'll add a little memo to self when updating themes where/what to modify in future after reading any latest updates here. 

Again thank you for your time. Recently I rebuilt one of my old pcs in closet and maxed out the cpu on it, threw a ssd in it and my old gaming 9800 card from years ago, primarily used as my home "cloud" with 5 internal 'WD blue' drives and 3 external drives; 2 usb & 1 esata. And gave Linux Mint a try and I really  enjoy it, super light, I absolutely loved that menu applet instead of stock. I've been a Slackware guy since late 90's around the 2.0.25 kernel, but Mint is one of the perfect distros for adding a ton of new life to an aging machine.

So thank you everyone, I greatly appreciate it and hope you all have a good weekend!
